operator/pkg/securedcluster/values/translation/translation.go

```diff
@@ -98,7 +99,12 @@ func (t Translator) translate(ctx context.Context, sc platform.SecuredCluster) (
 		return nil, err
 	}
 
-	v.AddChild("sensor", t.getSensorValues(sc.Spec.Sensor, scannerAutoSenseConfig))
+	scannerV4AutoSenseConfig, err := scanner.AutoSenseLocalScannerV4Config(ctx, t.client, sc)
```

Comment:
Just thinking loud here, feel free to resolve: Should we handle errors in a situation like this more forgiving, for example by picking a default value and continuing with the translation machinery?

Reply:
Good though, may be worth. Since I'm not aware of any issues with the implementation right now I think we should leave it as is for this PR and reassess at another point in time.